mappings/net/minecraft/client/render/block/entity/BannerBlockEntityRenderer.mapping

```diff
@@ -1,7 +1,20 @@
 CLASS net/minecraft/class_823 net/minecraft/client/render/block/entity/BannerBlockEntityRenderer
-	FIELD field_20810 field Lnet/minecraft/class_630;
-	FIELD field_20811 verticalBar Lnet/minecraft/class_630;
-	FIELD field_20812 topBar Lnet/minecraft/class_630;
+	FIELD field_20810 banner Lnet/minecraft/class_630;
```

Comment:
What is the banner of a banner?
```suggestion
	FIELD field_20810 field Lnet/minecraft/class_630;
```
this is like used by minecraft wiki banner page.

Reply:
Huh?

Field is too generic. The banner is literally a banner. It's a piece of cloth you hang on a wall of attach to a stand/pole.

```1.a: a piece of cloth attached by one edge to a staff and used by a leader (such as a monarch or feudal lord) as a standard```
```3 : a strip of cloth on which a sign is painted```

https://www.merriam-webster.com/dictionary/banner